### PR TITLE
P10 Added judging critera page

### PR DESCRIPTION
**Pull request by andrewdempsey2018**
Added basic "Judging Criteria" page with some placeholder text. Setup urls, view and navbar link to access the page.

### CODE REVIEW

File: .env_sample [hunk 1]
```diff
@@ -1,3 +0,0 @@
-DEVELOPMENT=1
```
Comment (stefdworschak) [inline .env_sample:1]: Why did you delete the file? Could you please add this back in as this is for all students? If you don't need it locally, you can delete it, but please don't commit the change.

File: home/templates/home/criteria.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+
+<H1>Judging Criteria for Hackaton</H1>
```
Comment (stefdworschak) [inline home/templates/home/criteria.html:6]: Please change this to lowercase `<h1>`

File: home/templates/home/criteria.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+<br>
```
Comment (stefdworschak) [inline home/templates/home/criteria.html:7]: In general I would avoid line breaks. Could you remove them and use bootstrap grid instead?

File: home/templates/home/criteria.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    <li>Content</li>
+    <li>Structure and Navigation</li>
+    <li>Visual Design</li>
+    <li>Functionality</li>
+    <li>Interactivity</li>
+    <li>Innovation</li>
+    <li>Overall Experience</li>
+</ul>
+<br>
+<h2><b>Content</b></h2>
```
Comment (stefdworschak) [inline home/templates/home/criteria.html:19]: Could you please replace the `<b>` tags with `<strong>` . Alternatively, you could create a custom css class called `bold` (or similar) and add the class to the `<h2>`.

File: home/templates/home/criteria.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+<p>Content is the information provided on the site. It is not just text, but music, sound, animation, or video –
+    anything that communicates a site’s body of knowledge. Good content should be engaging, relevant, and appropriate
+    for the audience; you can tell it’s been developed for the Web because it’s clear and concise and it works in the
+    medium. Good content takes a stand. It has a voice and a point of view. It may be informative, useful, or funny, but
+    it always leaves you wanting more.</p>
+<br>
+<h2><b>Structure and navigation</b></h2>
```
Comment (stefdworschak) [inline home/templates/home/criteria.html:27]: Same here (and for all of the instances of `<b>`)

File: home/views.py [hunk 6]
```diff
@@ -5,3 +5,8 @@ def index(request):
     """ A view to return the index page """
 
     return render(request, "home/index.html")
+
```
Comment (stefdworschak) [inline home/views.py:8]: We are following PEP8 standard where you would want 2 line breaks between functions or classes. Could you please add another line break here? If you are interested, you can read more here: https://www.python.org/dev/peps/pep-0008/

**Review by stefdworschak — CHANGES_REQUESTED** [review]
@andrewdempsey2018 this is a really good first pull request you can be proud of! Just a few minor changes that I would recommend. Once you made the changes, just tag me or Tim in the Pull Request again and we can approve it. For future use I would also recommend to flesh out the Pull Request message a little bit (e.g. add areas such as Description, Tests, Errors, Screenshots, etc.); we will also add a template shortly as well.

File: home/templates/home/criteria.html [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+                    download speed.
+                    The most functional sites also take into consideration those with special access needs. Good
+                    functionality makes the
+                    experience center stage and the technology invisible.</p>
+            </div>
+        </div>
+    </div>
+
+</div>
+{% endblock %}
```
Comment (TravelTimN) [inline home/templates/home/criteria.html:69]: @andrewdempsey2018 
needs a new line at the end of the file

**Comment by andrewdempsey2018** [comment]
@stefdworschak @TravelTimN 

User Story ID: #P10

Description of PR:
Added a basic Judging Criteria page with dummy information in lieu of a final writeup of criteria. Page contains four locations of information with headings on a 2x2 grid & is responsive for small screens. Home/urls.py reflects the new page as well as home/views.py

Tests:

- Page has been tested on desktop browsers Firefox and Chrome. It has also been tested on an emulated smartphone.

Screenshots:

Desktop Firefox test:

image

Desktop Chrome test:

image

Emulated smartphone test:

image

Know bugs/errors:

- Page needs actual content so actual judging critera needs to be written up.

Post review/additional changes:

- reinstated .env_sample after deleting it accidentally
- rewrote some tags with lower case to keep HTML code style consistent
- updated home>views.py to be in line with PEP8 standard
- added new line at the end of criteria.html

File: .env_sample (hunk 1 above)
Comment (TravelTimN) [reply .env_sample:1]: File added back in - resolving conversation

**Review by stefdworschak — APPROVED** [review]
LGTM

Just FYI - not sure if you are aware, but you can edit the PR message body. Also, usually if somebody leaves comments, you would use the Reply option for each comment and Resolve the conversation once the comment is addressed. Not a biggie though.

**Review by TravelTimN — APPROVED** [review]
LGTM - might get @stefdworschak to just check from his end.
Also - there might be some issues, since @JimLynx was working on the footer, and the view/url for this was different.
I do like @andrewdempsey2018 namespace better though, short and sweet.

**Comment by JimLynx** [comment]
> LGTM
> 
> Just FYI - not sure if you are aware, but you can edit the PR message body. Also, usually if somebody leaves comments, you would use the Reply option for each comment and Resolve the conversation once the comment is addressed. Not a biggie though.

Noted, thank you. Will ensure I do so in future.

**Comment by TravelTimN** [comment]
Actually, @andrewdempsey2018 - can you have a look at #74  and adjust the urls/views accordingly to match that of @JimLynx  please?
`judging_criteria` instead of just `criteria`, as this could eventually be expanded and have a different meaning for something else later. Thanks!

**Comment by andrewdempsey2018** [comment]
> judging_criteria

**Comment by andrewdempsey2018** [comment]
> 
> 
> LGTM
> 
> Just FYI - not sure if you are aware, but you can edit the PR message body. Also, usually if somebody leaves comments, you would use the Reply option for each comment and Resolve the conversation once the comment is addressed. Not a biggie though.

Noted, thanks for the tip.

**Review by stefdworschak — APPROVED** [review]
LGTM
